### PR TITLE
SAAS-526 migrate components to core UI

### PR DESCRIPTION
**Pull request by atymchuk**
This PR relates to:
- https://github.com/percona-platform/cicd/pull/30
- https://github.com/percona-platform/saas-ui/pull/86

### CODE REVIEW

File: .github/workflows/enable-docker-experimental.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -ex
+
+jq '.experimental = true' < /etc/docker/daemon.json > docker.json
+mv docker.json /etc/docker/daemon.json
+cat /etc/docker/daemon.json
```
Comment (nicolalamacchia) [inline .github/workflows/enable-docker-experimental.sh:7]: Why is this `cat` needed?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,34 @@
...
+    npm i
+    npm start
+```
+
+## Most useful commands
+
+- Start the development: `lerna run start --scope=@percona/platform-core` or just run `yarn start` from the package directory
+- Run tests: `lerna run test --scope=@percona/platform-core` or just run `yarn test` from the package directory
+- Build artifacts: `lerna run build --scope=@percona/platform-core` or just run `yarn build` from the package directory
+- Publish the package: `lerna run publish --scope=@percona/platform-core`
```
Comment (nicolalamacchia) [inline README.md:19]: This part should be updated or removed

**Review by nicolalamacchia — APPROVED** [review]

File: .github/workflows/enable-docker-experimental.sh (hunk 1 above)
Comment (atymchuk) [reply .github/workflows/enable-docker-experimental.sh:7]: To see if `experimental` is true.

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+insert_final_newline = true
+
+# Matches multiple files with brace expansion notation
+# Set default charset
+[*.{js,ts,tsx,go}]
+charset = utf-8
+
+[*.go]
+indent_style = tab
+indent_size = 4
```
[the remark this answers is not in the document]
Comment (atymchuk) [reply .editorconfig]: Removed

File: README.md (hunk 2 above)
Comment (atymchuk) [reply README.md]: Yes, forgot about that one. Removed.

File: .github/workflows/enable-docker-experimental.sh (hunk 1 above)
Comment (nicolalamacchia) [reply .github/workflows/enable-docker-experimental.sh:7]: Oh, you mean in the logs, right? Ok

**Review by nicolalamacchia — APPROVED** [review]

**Pull request merged**